backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/exceptions/DBException.java

```diff
@@ -25,6 +25,9 @@
  */
 public class DBException extends AdempiereException
 {
+
+	public static final String DB_ERROR_CODE_PREFIX = "DB - ";
```

Comment:
why having spaces into the error code?
Usually, error codes do not contain spaces; they are just less error-prone when spelled/written.

Reply:
removed spaces.